openlibrary/templates/books/add.html

```diff
@@ -71,7 +71,7 @@ <h1>$_("Add a book to Open Library")</h1>
         $if recaptcha and not ctx.user:
```

Comment:
This is a separate issue, but I have noticed it:  to prevent spam we should show a recaptcha to all new users on add book, like we do for edit books.

`recaptcha_exempt()` , which I believe is run here for getting the recaptcha will prevent it being displayed to admins and users who have been members for 2 years +

`$if recaptcha:` should be sufficient. Do you agree @mekarpeles ?